contracts/scripts/ci/find_slither_report_diff.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [[ "$#" -lt 4 ]]; then
+  >&2 echo "Generates a markdown file with diff in new issues detected by ChatGPT between two Slither reports."
+  >&2 echo "Usage: $0 <path-to-first-report> <path-to-second-report> <path-to-diff-report-output> <path-to-prompt> [path-to-validation-prompt]"
+  exit 1
+fi
+
+if [[ -z "${OPEN_API_KEY+x}" ]]; then
```

Comment:
If preferred, you can safely remove this as $openai_result below will fail with ::error:: log telling you that you haven't supplied the key and we won't be charged for the call.

Reply:
I prefer to have it, because it's running in strict mode and will error if env var is not set, when read. I prefer to return a more explicit error than what bash would.